### PR TITLE
PAY-5294 - iGaming Events API

### PR DESCRIPTION
**Pull request by ademianenko-sift**
## Purpose
Add support for iGaming Events API changes:
- extra fields for `$deposit` and `$withdrawal` Transactions
- new event type `$wager`

## Summary
Added support for the new fields and event type, lib version has been updated for a new release.

## Testing
Changes are covered with unit and integration tests.

## Checklist
- [x] The change was thoroughly tested manually
- [x] The change was covered with unit tests
- [x] The change was tested with real API calls (if applicable)
- [x] Necessary changes were made in the integration tests (if applicable)
- [x] New functionality is reflected in README

### CODE REVIEW

File: Sift/Schema/wager.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "required": [ "$type" ],
+  "properties": {
+    "$type": {
+      "type": "string",
+      "default": "$wager"
+    },
+    "$user_id": {
+      "type": [ "string", "null" ]
+    },
+    "$session_id": {
```
Comment (rsomavarapu-sift) [inline Sift/Schema/wager.json:14]: I don't see this in the PRD. Is this field needed?

**Review by rsomavarapu-sift — CHANGES_REQUESTED** [review]
Also, take a look at the warnings from GitHub actions.

File: Sift/Schema/wager.json (hunk 1 above)
Comment (ademianenko-sift) [reply Sift/Schema/wager.json]: Good catch, removed

File: Test/Test.cs [hunk 2]
```diff
@@ -2627,6 +2627,181 @@ public void TestWarnings()
...
+            };
+
+            Assert.Equal("https://api.sift.com/v205/events?abuse_types=legacy,payment_abuse&return_score=true",
+                          Uri.UnescapeDataString(eventRequest.Request.RequestUri!.ToString()));
+        }
+
+        [Fact]
+        public void TestWagerEvent()
+        {
+            var sessionId = "sessionId";
```
Comment (rsomavarapu-sift) [inline Test/Test.cs:2759]: Can you remove it here as well?

Comment (ademianenko-sift) [reply Test/Test.cs]: Done

**Review by rsomavarapu-sift — APPROVED** [review]

**Review by sbogolii-sift — APPROVED** [review]
LGTM

**Pull request merged**